### PR TITLE
feat(Priest-Shadow): Optimize Racial & Trinket Usage per Latest IV Guide

### PR DESCRIPTION
**Pull request by joshjau**
Summary Points:

Optimized racial ability usage:

- Synced offensive racials (Fireblood, Berserking, Blood Fury, Ancestral Call) with Power Infusion
- Added Light's Judgment with proper target count conditions
- Restricted Arcane Torrent to pre-combat only
- Added end-of-fight conditions for all racial

Enhanced trinket priority system:

- Added stack limit for Aberrant Spellforge (stops at 4 stacks)
- Improved High Speaker's Accretion timing before burst windows
- Optimized Spymaster's Web usage based on stack count and fight timing
- Added support for Mereldar's Toll
- Added target selection for Mad Queen's Mandate (low HP priority)
- Added stack requirements for Ovi'nax's Mercurial Egg
- Added Treacherous Transmitter support
- Refined default trinket behavior for better cooldown alignment

### CODE REVIEW

**Comment by Hekili** [comment]
Can you move any trinket-related auras into TWW/Items.lua?  I'm fine with the priority adjustments since it is all trinket logic related.

File: TheWarWithin/PriestShadow.lua [hunk 1]
```diff
@@ -1077,6 +1077,26 @@ spec:RegisterAuras( {
         duration = 5,
         max_stack = 1
     },
+    aberrant_spellforge = {
```
Comment (Hekili) [inline TheWarWithin/PriestShadow.lua:1080]: Move trinket auras to Items.lua (or Trinkets.lua, or whatever it is called) in TWW folder.

File: TheWarWithin/PriestShadow.lua [hunk 2]
```diff
@@ -2392,6 +2412,35 @@ spec:RegisterAbilities( {
             if talent.dark_evangelism.enabled then addStack( "dark_evangelism" ) end
         end,
     },
+
+    -- Add racial abilities
+    arcane_torrent = {
```
Comment (Hekili) [inline TheWarWithin/PriestShadow.lua:2417]: These belong in Hekili/Classes.lua (unless there is a functional distinction from what is there).  That may be the case for Arcane Torrent but probably not Light's Judgment.  Both should probably be on the Cooldowns toggle, as well.

File: TheWarWithin/Priorities/PriestShadow.simc [hunk 3]
```diff
@@ -157,5 +161,23 @@ actions.empowered_filler+=/mind_flay_insanity,cycle_targets=1,if=buff.mind_flay_
 ## actions.heal_for_tof+=/holy_nova,if=buff.rhapsody.stack=20&talent.rhapsody
 
 actions.trinkets+=/use_item,use_off_gcd=1,name=aberrant_spellforge,if=gcd.remains>0&buff.aberrant_spellforge.stack<=4
-actions.trinkets+=/use_item,name=spymasters_web,if=(buff.power_infusion.up&buff.spymasters_report.stack>=40&fight_remains>240)|(buff.power_infusion.up&buff.bloodlust.up&buff.spymasters_report.stack>=10)|buff.power_infusion.up&(fight_remains<120)|(fight_remains<=20|buff.dark_ascension.up&fight_remains<=60|buff.entropic_rift.up&talent.entropic_rift&fight_remains<=30)&!buff.spymasters_web.up
-actions.trinkets+=/use_items,if=(buff.voidform.up|buff.power_infusion.up|buff.dark_ascension.up|(cooldown.void_eruption.remains>10&trinket.cooldown.duration<=60))|boss&fight_remains<20
+# High Speaker's Accretion before burst windows
```
Comment (Hekili) [inline TheWarWithin/Priorities/PriestShadow.simc:164]: Extra spacing can be removed (reimport not needed).

**Review by Hekili — CHANGES_REQUESTED** [review]
Small edits.

**Comment by joshjau** [comment]
Thanks for the feedback. I will review this and make adjustments.

**Pull request closed without merging**